extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/PanacheCrudResourceInfo.java

```diff
@@ -1,10 +1,10 @@
-package io.quarkus.panache.rest.common.deployment;
+package io.quarkus.rest.data.panache.deployment;
 
 import org.jboss.jandex.ClassInfo;
 
 import io.quarkus.builder.item.MultiBuildItem;
 
-public final class PanacheCrudResourceBuildItem extends MultiBuildItem {
+public final class PanacheCrudResourceInfo extends MultiBuildItem {
```

Comment:
Any reason why this was renamed?

Reply:
Before it was already used by a processor which would extract separate bits of information and only pass what's needed to a specific method implementor. I made the method implementors more abstract now so processor wouldn't have to know too much details about them. Now it just passes this object to each implementor. So I assumed the name without "BuildItem" suffix would be more suitable. What do you think?

Reply:
It just feels weird to have being a build item and not being called that. What we usually do in similar cases is to have a build item that just holds a reference to the regular class. Then in the non-build-step code you can just pass around the regular class. The build step classes would of course the build items to communicate.

Reply:
Yeah that makes sense, I'll extract it to a separate class

Reply:
Done